game.py

```diff
@@ -1,18 +1,76 @@
+import cspclue
 import time
 import functools
+import numpy as np
 
 class Game(object):
+	'''
+	self.rooms - list of room card variables (except for room card in case file)
+	self.weapons - list of weapon card variables (except for weapon card in case file)
+	self.suspects - list of suspect card variables (except for suspect card in case file)
+
+	self.caseFileRoom - Room card variable in the case file
+	self.caseFileSuspect - Suspect card variable in the case file
+	self.caseFileWeapon - Weapon card variable in the case file
+	'''
 
 	def __init__(self):
 		'''
-		- Create all cards
-		- Assign case file (randomly)
+		- [DONE] Create all cards
+		- [DONE] Assign case file (randomly)
 		- Initialize agents
-		- assign cards to agents (Randomly)
+		- assign cards to agents (Randomly)			
 		- Choose a player to go first
 		'''
-		pass
+		self._init_cards()
+		self._make_case_file()
+
+	def _init_cards(self):
+		'''
+		Initialize all 21 cards
+		9 rooms, 6 weapons, 6 suspects
+		Return lists of each type
+		'''
+		#Init Rooms
+		self.rooms = []
+		rooms = ['Conservatory', 'Hall', 'Lounge', 'Dining Room', 'Kitchen', 'Ballroom', 'Billiard Room', 'Library', 'Study']
+		for room in rooms:
+			r = Card('Room', room, [room])
```

Comment:
When we are initing the cards, (even if they have a value) shouldn't their dom be all of the possible options, and have their curr_dom set as the assigned card?

@gracekumagai